### DISCUSSION TITLE
Auto-Prefix in language config

### DISCUSSION THREAD

**Discussion by MaxWainer** [Zrips/CMI#3530] [Feature Requests]
**Description of feature/improvement request:**
Little feature which auto-prefix all messages in language.yml.
---
### Detailed explanation of what it should do, when and in what situation: ###
In CMI so many commands and so many messages. And add to all '!prefix!' tag very tired and eat some precious part of time which could be spent on setting up more important and time-consuming plugins.
### Basic explanation as to why this feature is needed and/or useful to have: ###
Time saving.

P.S Sorry if I didn't fully reveal the essence. :/

**Comment by CloudeLecaw** [comment]
The current solution is fine. If you had an automatic prefix thus would avoid more serverowners that dont want any kind of prefixes in their usermeassages and stuffmassages. Many specialy smaller plugins fail in this point and use an automatic prerfix that often cant be changed. Personaly... If i see a plugin that is forcing me to have its prefix in all outputs, then im going to delet itand search for an other plugin. The current way in CMI is a fair deal. The owner can decide what language outputs use the prefix by Design it in the langfile and he has a prefix placeholder that is alsi defined in the langfile, or he can simly setup a unique prefix if he want by writing it into the chaoutputtline of the command
. A fair deal. I know the langfile is long but it wont get shorter anymore since cmi is getting more and more Features over the time. This is a normal behave of essential plugins
